libs/brackets-server/embedded-ext/file-extension/node/ImportDomain.js

```diff
@@ -105,7 +105,7 @@
                 console.log(stderr);
 
                 if (err) {
-                    return callback("Could not install " + packageName + " from " + wpmAddress + ", error: " + err);
+                    return callback("Could not install " + packageName + " from " + wpmAddress);
```

Comment:
I wonder if it would be cleaner if we use template literals in a case like this.
``` javascript
`Could not install ${packageName} from ${wpmAddress}`
```

Reply:
How about displaying this error on console.log? Having information why something fails is important I believe.

Reply:
@korneliakobiela I agree, I'll fix
@grzegorz-czajkowski OK, error will be printed on servers console so nothing will be revealed to user. I'll change

Reply:
done